modules/_labs/slider/react/lib/Slider.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from 'react';
+import {
+  borderRadius,
+  colors,
+  spacingNumbers as spacing,
+  type,
+} from '@workday/canvas-kit-react-core';
+import {focusRing} from '@workday/canvas-kit-react-common';
+
+import styled from '@emotion/styled';
+
+export interface SliderProps {
+  max: number;
+  min: number;
+  startValue: number;
+  step?: number;
+  useInputRange?: boolean;
+
+  onChange?: (newValue: number) => void;
+  onStartDrag?: (newValue: number) => void;
+  onKeyDown?: (newValue: number) => void;
+  onKeyUp?: (newValue: number) => void;
+  onEndDrag?: (newValue: number) => void;
+}
+
+const sliderContainerHeight = spacing.l;
+const sliderContainerWidth = 328;
+const sliderContainerMinWidth = 200;
+const sliderTrackRadius = 100;
+const sliderTrackHeight = 5;
+const sliderThumbHeight = spacing.s;
+const sliderThumbWidth = spacing.s;
+
+const Container = styled('div')({
+  position: 'relative',
+  width: sliderContainerWidth,
+  minWidth: sliderContainerMinWidth,
+  height: spacing.l,
+});
+
+const Interval = styled('div')({
+  fontSize: 'small',
+  color: colors.licorice300,
+  padding: spacing.xxs,
+});
+
+const SliderInput = styled('input')({
+  outline: 'none',
+  padding: 0,
+  margin: 0,
+  border: 'none',
+  width: sliderContainerWidth,
+  minWidth: sliderContainerMinWidth,
+  height: '100%',
+  cursor: 'pointer',
+  WebkitAppearance: 'none',
+  MozAppearance: 'none',
+  background: 'transparent',
+  position: 'absolute',
+  zIndex: 1,
```

Comment:
From what I can tell, `z-index` is needed because the `SliderInput` is defined before the `ProgressBarContainer` in the DOM. Is it possible to swap the definition of the 2 and remove this line?

I'm always cautious of `z-index` changes. There is often unintended side effects of changing it.